axelrod/fingerprint.py

```diff
@@ -210,28 +203,28 @@ def generate_data(self, interactions, coordinates):
         ----------
         interactions : dictionary
             A dictionary of the interactions of a tournament
-        probe_coords : list of tuples
+        coordinates : list of tuples
             A list of tuples of length 2, where each tuple represents a
             coordinate, eg. (x, y).
 
         Returns
         ----------
-        coord_scores : dictionary
+        coordinate_scores : dictionary
             A dictionary where the keys are coordinates of the form (x, y) and
             the values are the mean score for the corresponding interactions.
         """
         edge_scores = {key: np.mean([cfspt(i) for i in value]) for key, value in
                        interactions.items()}
 
-        coord_scores = {coord: None for coord in coordinates}
-        for index, coord in enumerate(coordinates):
-            if sum(coord) > 1:
+        coordinate_scores = {coord: None for coord in coordinates}
+        for index, coordinate in enumerate(coordinates):
+            if sum(coordinate) > 1:
                 edge = (1, index + 2)
             else:
                 edge = (0, index + 2)
```

Comment:
This block (lines 220-224) is a repeat of lines 109-115. That suggests to me there's a function here that needs to be pulled out to module level.